server/app/main.py

```
@@ -0,0 +1,45 @@
+            messages=body.get("messages", []),
+            model=body.get("model", "gpt-4o-mini"),
+            temperature=body.get("temperature", 1.0),
+            max_tokens=body.get("max_tokens", None),
+            stream=body.get("stream", False)
+        )
+        logger.info(f"Successfully processed request for model: {model}")
+        return response
+
+    except Exception as e:
+        logger.error(f"Error processing request: {str(e)}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)